distribution/pom.xml

```
@@ -433,34 +473,67 @@
           <plugin>
             <artifactId>jdeb</artifactId>
             <groupId>org.vafer</groupId>
-            <version>1.0</version>
+            <version>1.7</version>
             <executions>
               <execution>
                 <phase>package</phase>
                 <goals>
                   <goal>jdeb</goal>
                 </goals>
-                <configuration>
+		<configuration>
+		<skipPOMs>false</skipPOMs>
                   <deb>target/drill-${project.version}.deb</deb>
```

It will be better to keep all archives in one place. Since tarball archive is already in this directory, please use it for `rpm` and `deb` archives too.
```suggestion
                  <deb>distribution/target/apache-drill-${project.version}/apache-drill-${project.version}.deb</deb>
```

---

@vdiravka Thank you very much for review.  I try to address all your comments.  Please review.